corehq/apps/app_manager/templates/app_manager/v2/partials/case_config_shared.html

```diff
@@ -169,9 +169,21 @@ <h4 class="panel-title panel-title-nolink">
                             afterRender: $root.makePopover
                         }"></div>
                         <p class="help-block" data-bind="html: validate, visible: validate"></p>
-                        <div class="property-description"
-                          data-bind="text: $data.description, attr: {'data-content': $data.description}">
-                        </div>
+                        <!-- ko if: $data.description -->
+                        <inline-edit params="
+                            value: $data && $data.description,
```

Comment:
Yeah, this is a holdover from an older implementation, but is definitely unnecessary here. In general i like that construct in a situation like this though because it allows you to check if something exists inline. If it were possible for `$data` not to exist, that would have been the nicest and maybe only way to check